site/app/controllers/GlobalController.php

```diff
@@ -166,7 +166,7 @@ public function prep_course_sidebar(&$sidebar_buttons, $unread_notifications_cou
         if ($this->core->getConfig()->isPollsEnabled()) {
             $sidebar_buttons[] = new Button($this->core, [
                 "href" => $this->core->buildCourseUrl(['polls']),
-                "title" => "Polls",
+                "title" => "Submini Polls",
```

Comment:
Can we pull this into its own PR? This seems unrelated to the rest of the changeset, and so in-case we have to revert the rest of this PR, we wouldn't want to revert this change as well.

Reply:
I'll put this in another PR